editor/animation_track_editor.cpp

```diff
@@ -2696,13 +2696,15 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 					if (ap) {
 						NodePath npath = animation->track_get_path(track);
 						Node *nd = ap->get_node(ap->get_root())->get_node(NodePath(npath.get_concatenated_names()));
```

Comment:
We should store the result of `ap->get_node(ap->get_root())` in a tempoary `Node*` and also perform a null check on this part.

I checked and this is the only place in the file that performs either of these `get_node` calls so it's probably best to be safe. Not all AnimationPlayer nodes have their root set to `..`

Reply:
I have added the extra check and a little comment in the code to explain why we need it